Microsoft.NET.Build.Containers/Resources/Strings.resx

```diff
@@ -118,58 +118,58 @@
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <data name="AmazonRegistryFailed" xml:space="preserve">
-    <value>Request to Amazon Elastic Container Registry failed prematurely. This is often caused when the target repository does not exist in the registry.</value>
+    <value>CONTAINER1002: Request to Amazon Elastic Container Registry failed prematurely. This is often caused when the target repository does not exist in the registry.</value>
```

Comment:
Can you add comments for the localization team that let them know that the code shouldn't be localized? This is done with a comment of the form (example from the SDK codebase [here](https://github.com/baronfel/sdk/blob/7bf2f14fa73e720e6b390fc31ae35c139c54a991/src/Tasks/Common/Resources/Strings.resx#L122)):

```
<data name="AtLeastOneTargetFrameworkMustBeSpecified" xml:space="preserve">
    <value>NETSDK1001: At least one possible target framework must be specified.</value>
    <comment>{StrBegin="NETSDK1001: "}</comment>
  </data>
```